src/apify/_actor.py

```diff
@@ -50,6 +51,26 @@
 
 
 MainReturnType = TypeVar('MainReturnType')
+TFun = TypeVar('TFun', bound=Callable[..., Any])
+
+
+def _add_local_storage_error_hint(function: TFun) -> TFun:
```

Comment:
Couldn't we move the decorator to a dedicated private module, perhaps [apify._utils](https://github.com/apify/apify-sdk-python/blob/master/src/apify/_utils.py)?